web/viewer.css

```diff
@@ -1117,6 +1117,7 @@ a:is(.toolbarButton, .secondaryToolbarButton)[href="#"] {
   /*#endif*/
 
   .loadingInput:has(> &.loading)::after {
+    display: block;
```

Comment:
You need to add the same line to the findbar as well[1], since that spinner is broken now:
https://github.com/mozilla/pdf.js/blob/3d2eb36e272fa3e499aaab8d3613890f8e39e53b/web/viewer.css#L503-L505

---
[1] Please keep in mind that the "GENERIC" findbar is also being used in the *built-in* Firefox PDF when a PDF document is placed in e.g. an `iframe` element.